### PR TITLE
feat(docker): add docker-compose setup

### PR DESCRIPTION
**Pull request by fkm**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+version: '3'
+services:
+
+  document-merge-service:
+    image: adfinissygroup/document-merge-service@sha256:bfcfb01877d59bb2a73c3f21d435e6abde6f2b556a3f9c335238ecf9f892cadc
```
Comment (velrest) [inline docker-compose.yml]: Why are you referring to a specific sha here?

Comment (fkm) [reply docker-compose.yml]: We don't have versions for the document-merge-service.

But, as this is for development only, we could use `:latest` and have the developer debug possible problems with new versions.

I'm open for both approaches :smiley:

Comment (velrest) [reply docker-compose.yml]: I personally think `:latest` would probably be the better approach depending when the image gets built. If its on every master merge, you would have to update the `docker-compose.yml` file on each pr merged in the document-merge-service repo. But i will leave this up to you to decide :+1:

**Review by velrest — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (fkm) [reply docker-compose.yml]: Et voilà :sunglasses:

**Pull request merged**